src/geometries/ParametricGeometry.js

```diff
@@ -72,6 +77,16 @@ function ParametricBufferGeometry( func, slices, stacks ) {
 			var p = func( u, v );
 			vertices.push( p.x, p.y, p.z );
 
+			// approximate tangent plane vectors via central difference
+
+			pu.subVectors( func( u + EPS, v ), func( u - EPS, v ) );
+			pv.subVectors( func( u, v + EPS ), func( u, v - EPS ) );
```

Comment:
`func` normally returns a new instance of `Vector3`, see https://github.com/mrdoob/three.js/blob/dev/examples/js/ParametricGeometries.js